### DISCUSSION TITLE
使用useScroll, 有的电脑触底的时候arrivedState.bottom为true, 有的电脑却为false, 导致滚动加载失败

### DISCUSSION THREAD

**Discussion by wqz950424** [vueuse/vueuse#4251] [General]
使用useScroll, 有的电脑触底的时候arrivedState.bottom为true, 有的电脑却为false, 导致滚动加载失败，有大佬知道是为什么吗？

**Comment by ilyaliao** [comment]
有 example 嗎？

**Comment by wqz950424** [comment]
跟分辨率有关系，把触底距离加大就好了



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2025年02月07日 23:23 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [vueuse/vueuse] 使用useScroll, 有的电脑触底的时候arrivedState.bottom为true, 有的电脑却为false, 导致滚动加载失败 (Discussion #4251) |

有 example 嗎？
